src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.model.dto.request;
+
+public class UserRequestDto {
+    private String email;
+    private String password;
+    private byte[] salt;
```

Comment:
When the user login into your site, he must enter the salt?

Reply:
No